plugins/modules/device_credential_workflow_manager.py

```diff
@@ -2087,214 +2100,232 @@ def get_want_assign_credentials(self, AssignCredentials):
-                snmpV2cWrite_details = global_credentials.get("snmpV2cWrite")
-                if not snmpV2cWrite_details:
+                global_snmp_v2c_write_details = global_credentials.get("snmpV2cWrite")
+
+                if not global_snmp_v2c_write_details:
                     self.msg = "Global snmp_v2c_write Credential is not available"
                     self.status = "failed"
                     return self
-                snmpV2cWriteDetail = None
-                if snmpV2cWriteId:
-                    snmpV2cWriteDetail = get_dict_result(snmpV2cWrite_details, "id", snmpV2cWriteId)
-                    if not snmpV2cWriteDetail:
+                snmp_v2c_write_detail = None
+
+                if snmp_v2c_write_id:
+                    snmp_v2c_write_detail = get_dict_result(global_snmp_v2c_write_details, "id", snmp_v2c_write_id)
+                    if not snmp_v2c_write_detail:
                         self.msg = "The ID of the snmp_v2c_write credential is invalid."
                         self.status = "failed"
                         return self
-                elif snmpV2cWriteDescription:
-                    for item in snmpV2cWrite_details:
-                        if item.get("description") == snmpV2cWriteDescription:
-                            snmpV2cWriteDetail = item
-                    if not snmpV2cWriteDetail:
+                elif snmp_v2c_write_description:
+                    for item in global_snmp_v2c_write_details:
+                        if item.get("description") == snmp_v2c_write_description:
+                            snmp_v2c_write_detail = item
+
+                    if not snmp_v2c_write_detail:
                         self.msg = "The username and description of the snmp_v2c_write credential are invalid."
                         self.status = "failed"
                         return self
+
                 if current_ccc_version_as_int <= self.get_ccc_version_as_int_from_str("2.3.5.3"):
-                    want.get("assign_credentials").update({"snmpV2WriteId": snmpV2cWriteDetail.get("id")})
+                    want.get("assign_credentials").update({"snmpV2WriteId": snmp_v2c_write_detail.get("id")})
                 else:
                     want.get("assign_credentials").update({
-                        "snmpv2cWriteCredentialsId": {"credentialsId": snmpV2cWriteDetail.get("id")}
+                        "snmpv2cWriteCredentialsId": {"credentialsId": snmp_v2c_write_detail.get("id")}
                     })
 
-        https_read = AssignCredentials.get("https_read")
+        https_read = assign_credentials.get("https_read")
         if https_read:
-            httpReadId = https_read.get("id")
-            httpReadDescription = https_read.get("description")
-            httpReadUsername = https_read.get("username")
-            if httpReadId or httpReadDescription and httpReadUsername:
+            http_read_id = https_read.get("id")
```

Comment:
Can you check this part with Muthu Rakesh? The local variables should be https_* rite?

http**s**_read_id = https_read.get("id")
http**s**_read_description = https_read.get("description")
http**s**_read_username = https_read.get("username")

Reply:
changed